pajbot/managers/emote.py

```diff
@@ -95,14 +96,14 @@ def channel_emotes(self):
 
     def load_channel_emotes(self):
         streamer = StreamHelper.get_streamer()
-        self.tier_one_emotes, self.tier_two_emotes, self.tier_three_emotes = self.legacy_api.get_channel_emotes(
-            streamer
+        self.tier_one_emotes, self.tier_two_emotes, self.tier_three_emotes = self.twitchemotesapi.get_channel_emotes(
+            StreamHelper.get_streamer_id(), streamer
```

Comment:
Getting streamer and streamer ID should be in the same fashion so either inline the `streamer` variable or extract the function call to a variable